crates/gen-js/src/lib.rs

```diff
@@ -140,6 +140,16 @@ impl Js {
         }
     }
 
+    fn unwrap_nullable_option<'a>(&self, iface: &'a Interface, ty: &Type) -> Option<&'a Type> {
```

Comment:
Instead of duplicating the logic fo `is_nullable_option` could there be only one method here? (probably this one) and callers of `is_nullable_option` switch to this method with a `.is_some()` or something like that?

Also I think this may be better named something like `get_nullable_option` since "unwrap" typically has panicking connotations in Rust.

Reply:
I moved the methods into the Interface struct, which opens the door for more refactoring in a later PR.